client/src/app/board/page.tsx

```diff
@@ -1,16 +1,21 @@
 "use client";
 
 import DeleteDialog from "@/components/delete-dialog";
+import EditBoardDialog from "@/components/edit-board-dialog";
 
 export default function Board() {
   const handleDelete = () => {
     console.log("Deleted board");
   };
+  const handleEdit = () => {
+    console.log("Edited board");
+  };
 
   return (
     <div className="flex flex-col gap-5 p-5 w-screen h-screen">
       <div className="flex flex-row gap-5">
         <div className="text-3xl">Insert board name here</div>
+        <EditBoardDialog componentName="board" handleEdit={handleEdit} />
```

Comment:
We only need to pass in the componentName and handler if the dialog component is a reusable component (ie. EditDialog), so we can either make this a reusable EditDialog component or make it specifically for editing the board.

Reply:
Hmm. I'm leaning towards making it specifically for the board but I'm not sure how that should look. In my mind it'd be nice for the component to have information about itself and I'm thinking that can be passed down the tree from the board page.

Reply:
Maybe make it similar to the create board dialog since we have the same fields. You can put the componentName and handleEdit in its own component instead of passing it down as props

Reply:
Ok changes made.